db/migrate/20150804093148_add_thermometer_to_campaign_pages.rb

```diff
@@ -0,0 +1,5 @@
+class AddThermometerToCampaignPages < ActiveRecord::Migration
+  def change
+    add_column :campaign_pages, :thermometer, :boolean, default: false
```

Comment:
This is just for demo, right? We clearly don't want to make every field of every widget a field of the campaign page. But to get to a real MVP including petitions, we're going to need to either add more fields in this hacky way, or we're going to actually need to start building out the plugin idea.